_component_design/buttons-bar.md

```diff
@@ -0,0 +1,181 @@
+
+## Specifications
+
+Use the following specifications when you construct a buttons bar.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A buttons bar contains the following elements:
+
+- **Simple button (required)**: The simple button represents the primary action that the user can take. A buttons bar can contain multiple simple buttons.
+- **Drop-down button (required)**: When the user clicks the drop-down buttons, additional actions appear on a drop-down menu.
+- **Divider (required)**: The divider separates the simple button and the drop-down button.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Buttons bar composition"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons-bar/buttonbar-composition.png" width="284"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct a buttons bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Buttons bar style"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons-bar/buttonbar-style.png" width="404"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Use these spacing guidelines when you construct a buttons bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Buttons bar spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons-bar/buttonbar-spacing.png" width="137"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Primary button bar
+
+You can use primary buttons in a buttons bar. Do not include tertiary buttons in a buttons bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Primary buttons bar"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons-bar/buttonbar-variations-primary.png" width="137"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
```

Comment:
Do you mean 'present' instead of 'prevent'? Let's chat with @palindromebart and @racksmangonzalez about including the visual you've described. Thoughts, all?

Reply:
yep - present

Reply:
Decided in bowling to articulate that multiple buttons in a button bar is allowed, but we decided to not include a visual.